lib/qif/writer.rb

```diff
@@ -61,11 +62,17 @@ def initialize(io, type = 'Bank', format = 'dd/mm/yyyy')
     
     # Add a transaction for writing
     def <<(transaction)
-      @transactions << transaction
+      case transaction.class.to_s
```

Comment:
I think that we should support multiple accounts then, if someone has the time to update this PR?